### PR TITLE
fix(query): uncomment cloud formation's test sample

### PR DESCRIPTION
**Pull request by lipeavelar**
**Proposed Changes**
- Uncomment `IAM Access Analyzer Not Enabled` negative test sample

Signed-off-by: Felipe Avelar <felipe.avelar@outlook.com>
I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/cloudFormation/aws/iam_access_analyzer_not_enabled/test/negative1.yaml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-#            - Property: "resource"
-#              Contains:
-#                - "arn:aws:s3:::docs-bucket"
-#                - "arn:aws:s3:::clients-bucket"
+         RuleName: ArchivePublicS3BucketsAccess
+         Filter:
+           - Property: "resource"
+             Contains:
+               - "arn:aws:s3:::docs-bucket"
+               - "arn:aws:s3:::clients-bucket"
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/iam_access_analyzer_not_enabled/test/negative1.yaml:24]: Hello, @lipeavelar!

Thank you so much for continuing to improve KICS 🚀 

The sample is an invalid YAML. Sorry. It should be:

```suggestion
AWSTemplateFormatVersion: 2010-09-09
Resources:
 Analyzer:
   Type: "AWS::AccessAnalyzer::Analyzer"
   Properties:
     AnalyzerName: MyAccountAnalyzer
     Type: ACCOUNT
     Tags:
       - Key: Kind
         Value: Dev
     ArchiveRules:
        -
          # Archive findings for a trusted AWS account
          RuleName: ArchiveTrustedAccountAccess
          Filter:
            -
              Property: 'principal.AWS'
              Eq:
                - '123456789012'
        -
          # Archive findings for known public S3 buckets
          RuleName: ArchivePublicS3BucketsAccess
          Filter:
            -
              Property: 'resource'
              Contains:
                - 'arn:aws:s3:::docs-bucket'
                - 'arn:aws:s3:::clients-bucket'
```

However, with this change, the query will return results on `file.json`. Can you delete this file, please? (It is not relevant for the use case)

P.S. Please, update the branch with the master

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/cloudFormation/aws/iam_access_analyzer_not_enabled/test/negative1.yaml (hunk 1 above)
Comment (lipeavelar) [reply assets/queries/cloudFormation/aws/iam_access_analyzer_not_enabled/test/negative1.yaml:24]: There is no problem no removing this file, but there is a reason to remove negative sample? If this is what is wanted, we remove it ^^

Comment (rafaela-soares) [reply assets/queries/cloudFormation/aws/iam_access_analyzer_not_enabled/test/negative1.yaml:24]: I was referring to the `file.json`. Sorry for the confusion 😆

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**